docs/src/modules/operations/pages/services/deploy-service.adoc

```diff
@@ -216,6 +217,15 @@ After editing the service descriptor (e.g., `my-service.yaml`), redeploy it with
 akka service apply -f my-service.yaml
 ----
 
+=== Editing in place the service descriptor
```

Comment:
or is it better English as "Editing the service descriptor in place" ?

Reply:
I think you are right. Asked to https://www.deepl.com/en/write and agrees with you.

Reply:
done